src/ninja.cc

```diff
@@ -1163,7 +1163,7 @@ int main(int argc, char** argv) {
 #if defined(_MSC_VER)
   // Set a handler to catch crashes not caught by the __try..__except
   // block (e.g. an exception in a stack-unwind-block).
-  std::set_terminate(TerminateHandler);
+  set_terminate(TerminateHandler);
```

Comment:
This one was actually necessary. It was added in #1003 to fix MSVS2015, and #1110 now added it back again.